scripts/scil_visualize_multi_shell.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+
+import argparse
+import logging
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.viz.sampling_scheme import (build_ms_from_shell_idx,
+                                        build_shell_idx_from_bval,
+                                        plot_each_shell,
+                                        plot_proj_shell)
+
+
+DESCRIPTION = """
+Vizualisation for sampling schemes.
+Only supports .caru, .txt (Philips), .dir or .dvs (Siemens), .bvecs/.bvals
+and .b (MRtrix).
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
```

Comment:
This arg parser could be made clearer with the help of groups. Also, the ordering of the arguments is inconsistent